### PR TITLE
Fix data chunks concatenation

### LINKED ISSUE #3: Chunks concatenation breaks code syntax
The [line](https://github.com/cr0cK/rev-data/blob/master/index.js#L73) `const data = dataParts.join('\n');` causes pretty harsh code breaks like:

```
@media screen and (min-width: 40em) {
  .Header-root_1ASGC {
    height: 5
0px;
  }
}
```

Since we can't predict where the stream chunks will start/end.

### PR DESCRIPTION
**Pull request by pascalduez**
- Prevent arbitrary code breaks
- Fixes #3

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,7 +24,6 @@ commander
 process.stdout.write('Waiting data from stdin...\n');
 
 process.stdin.resume();
-process.stdin.setEncoding('utf8');
```
Comment (pascalduez) [inline index.js:27]: This had the effect of converting Buffers to Strings already.

File: index.js [hunk 2]
```diff
@@ -64,13 +63,13 @@ if (commander.assetmanifest) {
...
-process.stdin.on('data', function(data) {
+process.stdin.on('data', function (chunk) {
   clearTimeout(timeID);
-  dataParts.push(data);
+  dataParts.push(chunk);
 });
 
 process.stdin.on('end', function() {
-  const data = dataParts.join('\n');
+  const data = Buffer.concat(dataParts).toString();
```
Comment (pascalduez) [inline index.js:72]: Default encoding for `toString()` is `utf8` so all good :)

**Comment by yanndinendal** [comment]
:+1:

**Comment by cr0cK** [comment]
Good catch! Finally, the bug was there...

**Pull request merged**